test/extended/security/supplemental_groups.go

```diff
@@ -28,19 +32,22 @@ const (
 var _ = g.Describe("[sig-node] supplemental groups", func() {
 	defer g.GinkgoRecover()
 
-	var (
-		oc = exutil.NewCLI("sup-groups")
-		f  = oc.KubeFramework()
-	)
+	oc := exutil.NewCLIWithPodSecurityLevel("sup-groups", admissionapi.LevelBaseline)
+	ctx := context.Background()
 
 	g.Describe("Ensure supplemental groups propagate to docker", func() {
-		g.It("should propagate requested groups to the container [Local][apigroup:user.openshift.io][apigroup:security.openshift.io]", func() {
+		g.It("should propagate requested groups to the container [apigroup:security.openshift.io]", func() {
 
 			fsGroup := int64(1111)
 			supGroup := int64(2222)
 
+			projectName := oc.Namespace()
+			sa := createServiceAccount(ctx, oc, projectName)
+			supSubject := fmt.Sprintf("system:serviceaccount:%s:%s", projectName, sa.Name)
+			createSupRoleOrDie(ctx, oc, sa)
 			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
-				_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
+				// sa serves as a subject instead of the user
+				_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", supSubject).Output()
```

Comment:
IIRC `-z <SA name> -n <namespace>` is what should be used here, right?

Reply:
both of these options should work, but yes this makes the intent more clearer